### PR TITLE
Appservice insights

### PR DESCRIPTION
**Pull request by JFolberth**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by JFolberth** [comment]
Are not all the API providers available on the USGov side? I can't find a list of what is and isn't enabled.

File: 201-web-app-loganalytics/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+      },
+      "resources": [
+        {
+          "apiVersion": "2019-08-01",
+          "name": "appsettings",
+          "type": "config",
+          "dependsOn": [
+            "[resourceId('Microsoft.Web/Sites', variables('WebsiteName'))]",
+            "Microsoft.ApplicationInsights.AzureWebSites",
+            "[resourceId('microsoft.insights/components/', variables('appInsightName'))]"
```
Comment (bmoore-msft) [inline 201-web-app-loganalytics/azuredeploy.json:89]: you have a variable defined for this - so let's be consistent and use it everywhere or use the resourceId function everywhere - either is fine, but not both.

File: 201-web-app-loganalytics/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+  },
+  "variables": {
+    "appServicePlanName": "[toLower(concat('asp-', parameters('appName')))]",
+    "webSiteName": "[toLower(concat('wapp-', parameters('appName')))]",
+    "appInsightName": "[toLower(concat('appi-',parameters('appName')))]",
+    "appInisghtsReference": "[concat('microsoft.insights/components/',variables('appInsightName'))]",
+    "logAnalyticsName": "[toLower(concat('la-',parameters('appName')))]"
+  },
+  "resources": [
+    {
```
Comment (bmoore-msft) [inline 201-web-app-loganalytics/azuredeploy.json:40]: check the BP guide for sort order of properties, each resource seems to vary a bit:

https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Review by JFolberth — COMMENTED** [review]
Updated where I could.  Please review commons on the requested AppInsights reference changes.

**Pull request merged**